body.tex

```diff
@@ -692,7 +700,98 @@ \subsection{Reporting Cycle}
 Rubin will show progress on all L1 milestones and any L2 milestones called out in the POP.
 
+% ======================================================================
+\section{Annual Budget Planning}
+\label{sec:annual-cycle}
+
+Rubin Operations are funded at approximately equal levels by the NSF and DOE. NSF funding is requested by NOIRLab through its 5-year renewal proposal process, and annual budgets submitted as part of the NOIRLab Program Operations Plan (POP). DOE funding is requested by SLAC in an annual Field Work Proposal (FWP). In this section we describe the annual cycle of budget development and spending planning.
+
+This section defines and formalizes a ground-up process to
+prioritize the annual scope of the program,
+review and scrub budgets and performance,
+and develop, using Rubin's planning tools, a consolidated and detailed, cost estimate informed by the schedule milestones.
+
+It also describes our approach to risk-aware budgeting, and a process for reviewing and approving requests beyond baseline targets.
+
+
+\subsection{The annual budget planning cycle}
+
+Figure~\ref{fig:annual-cycle} illustrates the phases of Rubin Operations budget development and spending planning throughout the year, as laid out in Section~\ref{sec:annual-planning-timeline}.
+In April, following several months collecting high level and external budget information and implementing changes requested during the previous summer, the Director's Office sets major (Level 1 and some Level 2) milestones, and issues departmental budget guidance for the next two fiscal years.
+(Ideally, the Joint agency Operations status Review (JOR) would be held in early April, to provide external expert oversight of the plan.)
+The major milestones and budget are inserted into the draft NOIRLab POP and SLAC FWP.
+(Only the budget for the coming fiscal year is required for the POP.)
+
+In May, the Departments and Teams hold a series of meetings to set their minor (remaining Level 2 and Level 3) milestones, and ``scrub'' their budgets, milestones, and risks, in order to propose their next-year spending plans (to the Director's Office) by the end of May.
+Each department is given a ``sandbox'' workbook to work up during this period.
+This sandbox is initialized with the department's labor and non-labor plans (that were used in setting the budget), the current staffing plan, the progress against milestones in the current fiscal year, and the major milestones for the coming fiscal year.
+They are then invited to ``scrub'' their plans, by working this sandbox up with analysis of recent performance, and any changes they want to propose for the next fiscal year, based on the minor milestones and associated activities they want to prioritize.
+They are also invited to scrub their part of the risk register, update their risk response plans in concert with their planned activities, and anticipate any requests beyond target they may need to make.
```

Comment:
Good catch: I think the "theys" are justified here, because "we" in this document refers to Rubin Ops L0 and L1 mgmt plus program ops (ie the coauthors of the doc), not L2 mgmt (Team Leaders) more generally. However, the passive voice is mismatched with the rest of the document. I will fix that.